src/Symfony/Bridge/Twig/Extension/AssetExtension.php

```diff
@@ -21,10 +22,12 @@
 class AssetExtension extends \Twig_Extension
 {
     private $packages;
+    private $preloadManager;
 
-    public function __construct(Packages $packages)
+    public function __construct(Packages $packages, PreloadManagerInterface $preloadManager = null)
```

Comment:
The default `PreloadManagerInterface` implementation is simple and does not have dependencies, so I suppose it does not make sense to have a proper runtime.